servicetalk-http-api/src/main/java/io/servicetalk/http/api/ContentCodings.java

```diff
@@ -51,7 +51,7 @@ public static StreamingContentCodec identity() {
 
     /**
      * Returns a GZIP based {@link StreamingContentCodec} backed by {@link java.util.zip.Inflater}.
-     * The max allowed payload size for this codec is 2Mib.
+     * The max allowed payload size for this codec is 16Mib.
```

Comment:
We usually do not clarify default values in javadoc. Saving us some room the change them later. Consider removing it.